ewatercycle/parametersetdb/config.py

```diff
@@ -36,7 +38,6 @@ def __init__(self, source: str):
         """
         self.source = source
 
-    @abstractmethod
```

Comment:
Without `@abstractmethod `decorator an instance of the class can be made which is unwanted. Why was it removed?

Reply:
`mypy` complained that you cannot have an @abstractmethod on a class that derives from `ABC`. From my understanding, a class either derives from `ABC`, or it uses `@abstractmethod` to mark methods as abstract. But please correct me if I'm wrong.

Reply:
For me having both ABC and decorator is needed to get an error on construction of the abstract class. Also mypy does not complain for me.

Reply:
Alright, I will have another look at this 👍

Reply:
Okay, I have reverted the change, but I don't know how to fix these issues:
```
ewatercycle/parametersetdb/__init__.py:55: error: Cannot instantiate abstract class 'AbstractCopier' with abstract attributes '__init__' and 'save'
ewatercycle/parametersetdb/__init__.py:56: error: Cannot instantiate abstract class 'AbstractConfig' with abstract attributes '__init__', 'config' and 'save'
```